pages/about.html

```diff
@@ -4,9 +4,58 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="/styles/style.css" rel="stylesheet">
```

Comment:
Note that this `href` path won't load the stylesheet you have in `/stylesheets/stylesheet1.css`. As-written, it would try to load a file called "style.css" from the current directory, so from the project root, it'd be /pages/styles/style.css, which doesn't exist. If you wanted to load your stylesheet from its actual location, you'd use `<link href="../stylesheets/stylesheet.css" rel="stylesheet>`